harness/determined/pytorch/_lightning/README.md

```diff
@@ -0,0 +1,168 @@
+# PTL Adapter
```

Comment:
Is this `README.md` ready to be merged? It looks more like a development note rather than being useful to users.

Reply:
no, that's right. it's here mostly for reviewers benefit I'll take it out